kernel/environ.ml

```diff
@@ -675,11 +679,11 @@ let remove_hyps ids check_context check_value ctxt =
 open Retroknowledge
 
 (* lifting of the "get" functions works also for "mem"*)
-let retroknowledge f env =
-  f env.retroknowledge
+let retroknowledge env =
```

Comment:
This function doesn't seem useful anymore, I would deprecate it in favor of the projection.